src/applications/personalization/search-representative/config/chapters/search/search.js

```diff
@@ -1,4 +1,5 @@
 import SearchRepresentativeWidget from './searchRepresentativeWidget.jsx';
+import searchRepresentativeReviewWidget from './searchRepresentativeReviewWidget.jsx';
```

Comment:
Can we remove the 'jsx' from these 2 imports?

Reply:
Yep, definitely. Just did this.